.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
even though you added in gitignore you need to run a remove command for the ones that are part of this commit as @Kgrah

Reply:
Googling how to do this now